eth_test_parser/src/deserialize.rs

```diff
@@ -138,10 +138,27 @@ pub(crate) struct PreAccount {
     pub(crate) storage: HashMap<U256, U256>,
 }
 
+#[derive(Deserialize, Debug)]
+#[serde(rename_all = "camelCase")]
+pub(crate) struct AccessList {
+    pub(crate) address: Address,
+    pub(crate) storage_keys: Vec<U256>,
+}
```

Comment:
Just looking at the docs for the test format:

```json
"data": "0xDA7A",
"accessList": [
    {
        "address": "0x0000000000000000000000000000000000000101",
        "storageKeys": [0x60A7, 0xBEEF]
    },
    {
        "address": "0x0000000000000000000000000000000000000102"
    }
]
```
It looks like `storageKeys` is allowed to be missing, so we probably need to wrap that in an `Option` or similar (or maybe just a `DefaultOnNull`).